scripts/utils.py

```diff
@@ -3,6 +3,126 @@
 from constants import REQUEST_TIMEOUT, HEADER
```

Comment:
Is the code executed in local with verbose=true for reviewing `debug.log` that no sensitive data getting logged ?